### PR TITLE
DRAFT fix: 45076 optional boolean types props fixed

### PR DESCRIPTION
**Pull request by romesh87**
https://atera.tpondemand.com/entity/45076-fe-engage-ui-fix-optional-boolean

- All proptype?: boolean is changed to proptype: boolean, default value is now 'false'
- All components fixed
- All tests fixed
- All examples fixed

WARNING: this fix introduces breaking changes
DO NOT MERGE YET

UPDATE: all test issues fixes

### CODE REVIEW

**Comment by weegrom** [comment]
@romesh87 does this need a code review?

**Comment by weegrom** [comment]
@romesh87 does this need a code review?

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -179,6 +179,15 @@ export function buttonFrom(
       onClick={onAction}
       {...action}
       {...overrides}
+      disabled={false}
```
Comment (CaptainSadaf) [inline src/components/Button/Button.tsx:182]: @romesh87 Could you add default value in props params. What If I need to set default value of disable=true from external props?
const button = ({
  disabled = false,
}: Props)

Comment (romesh87) [reply src/components/Button/Button.tsx:182]: @CaptainSadaf these boolean props are not passed to THEMEDBUTTON component by default on the master branch

**Comment by romesh87** [comment]
opened new PR for this task

**Pull request closed without merging**